package.json

```diff
@@ -54,6 +53,7 @@
     "node": ">=16"
   },
   "dependencies": {
+    "@typescript-eslint/types": "^7.1.0",
```

Comment:
Why is this being added to `dependencies` instead of `devDependencies`?

Reply:
I added it to `dependencies` because it is referenced in the public types in `jsdoccomment.js` [here](https://github.com/typhonrt/jsdoccomment/blob/main/src/jsdoccomment.js#L15-L18) and [here](https://github.com/typhonrt/jsdoccomment/blob/main/src/jsdoccomment.js#L43).

This does also concern `estree` / `eslint` as well which has import types used in `jsdoccomment.js` that are not added to  `dependencies`. So the culprit would be refining the use cases around `jsdoccomment.js`. Technically `@types/eslint` / `@types/estree` should also be added to `dependencies`.

Reply:
There are a good number of bundle-size-conscious folks looking at `eslint-plugin-jsdoc`, but I presume the file size ought to be quite small on these type packages, and it does make sense that consumers shouldn't need to be concerned with dependencies.